docs/identity/role-based-access-control/permissions-reference.md

```diff
@@ -1797,7 +1797,7 @@ Assign the IoT Device Administrator role to users who need to do the following t
 > | Actions | Description |
 > | --- | --- |
 > | microsoft.directory/certificateBasedDeviceAuthConfigurations/create | Create Certificate Authorities configurations for IoT Device trust and authentication |
-> | microsoft.directory/certificateBasedDeviceAuthConfigurations/credentials/update | Update crendential related properties on certificate authority configurations for Internet of Things (IoT) device trust and authentication |
```

Comment:
Please revert this change

Reply:
Reverted. This has correct namespace and methods but the description has the spelling. Can you take this up the tree so they fix it ?